pkg/agent/controller/agent.go

```diff
@@ -3,18 +3,18 @@ package controller
 import (
 	"fmt"
 
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/util/retry"
```

Comment:
Minor thing but why were these moved up here separated from the other `k8s.io` imports?

Reply:
Done.